rest-server/test/jobSubmission.js

```diff
@@ -4,7 +4,7 @@
 // MIT License
 //
 // Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// documentation files (the 'Software'), to deal in the Software without restriction, including without limitation
```

Comment:
do not need to change?

Reply:
Sorry, I replace " to ' by mistake. I will revert this change